spec/support/vcr_cassettes/mobile/profile/v2/address_transaction_status.yml

```diff
@@ -0,0 +1,65 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: <VETS360_URL>/contact-information-hub/contact-information/v2/addresses/status/1bfcadca-fbfa-427c-8da2-96da9111f16b
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - Vets.gov Agent
+      Content-Type:
+      - application/json
+      Cufsystemname:
+      - VETSGOV
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: ''
+    headers:
+      X-Oneagent-Js-Injection:
+      - 'true'
+      - 'true'
+      Server-Timing:
+      - dtRpid;desc="1627627539", dtSInfo;desc="0"
+      - dtRpid;desc="955685449", dtSInfo;desc="0"
+      Vaprofiletxauditid:
+      - 96012705-cb62-4b16-be1a-384100869433
+      X-Content-Type-Options:
+      - nosniff
+      X-Xss-Protection:
+      - '0'
+      Cache-Control:
+      - no-cache, no-store, max-age=0, must-revalidate
+      Pragma:
+      - no-cache
+      Expires:
+      - '0'
+      X-Frame-Options:
+      - DENY
+      Content-Security-Policy:
+      - 'default-src ''self'' ''unsafe-eval'' ''unsafe-inline'' data: filesystem:
+        about: blob: ws: wss:'
+      Date:
+      - Thu, 12 Sep 2024 23:55:41 GMT
+      Referrer-Policy:
+      - no-referrer
+      Content-Type:
+      - application/json
+      Transfer-Encoding:
+      - chunked
+      Strict-Transport-Security:
+      - max-age=16000000; includeSubDomains; preload;
+    body:
+      encoding: UTF-8
+      string: '{"txAuditId":"0ea91332-4713-4008-bd57-40541ee8d4d4","status":"COMPLETED_SUCCESS","txStatus":"COMPLETED_SUCCESS","txType":"PUSH","txInteractionType":"ATTENDED","txPushInput":{"sourceDate":"2024-08-28T18:51:06Z","originatingSourceSystem":"VETSGOV","sourceSystemUser":"123498767V234859","addressId":15035,"addressPOU":"RESIDENCE","addressLine1":"1494
+        Martin Luther King Rd","cityName":"Fulton","zipCode5":"38843","country":{}},"txOutput":[{"createDate":"2018-09-05T16:58:15Z","updateDate":"2024-09-12T16:06:11Z","txAuditId":"0ea91332-4713-4008-bd57-40541ee8d4d4","sourceSystem":"VETSGOV","sourceDate":"2024-08-28T18:51:06Z","originatingSourceSystem":"VETSGOV","sourceSystemUser":"123498767V234859","effectiveStartDate":"2024-08-28T18:51:06Z","vaProfileId":1,"addressId":15035,"addressType":"Domestic","addressPOU":"RESIDENCE","addressLine1":"1494
```

Comment:
1494 Martin Luther King Rd  is not the correct address. All addresses built with :va_profile_v3_address should return this [address](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/factories/va_profile/v3/addresses.rb#L5). Please update:

va_profile/v2/contact_information/put_address_transaction_status
va_profile/v2/contact_information/put_address_success
va_profile/v3/contact_information/address_transaction_status
va_profile/v3/contact_information/get_address_status_incomplete
va_profile/v3/contact_information/put_address_initial
va_profile/v3/contact_information/get_address_status_complete
va_profile/v3/contact_information/get_address_status_complete_2
va_profile/v3/contact_information/get_address_status_error